packages/cli/generators/openapi/spec-loader.js

```diff
@@ -42,7 +42,14 @@ async function loadSpec(specUrlStr, {log, validate} = {}) {
       },
     });
   } else {
-    spec = await parser.dereference(spec);
+    try {
+      spec = await parser.dereference(spec);
+    } catch (error) {
+      // If returns http unauthorized error, ignore resolving external ref$ pointer
+      if (error?.message?.includes('HTTP ERROR 403')) {
```

Comment:
@sohaibahsan007, sorry that I should've caught it earlier. Would we be able to use error code instead of looking for particular string in the error message?

Reply:
@dhmlau  We have updated the string comparison with instanceof. Now we check for the type of error which is ResolverError.